test/conformance/ingress/path_test.go

```diff
@@ -210,27 +213,40 @@ func TestPathAndPercentageSplit(t *testing.T) {
 	defer cancel()
 
 	const (
-		total     = 100
+		total     = 1000
 		totalHalf = total / 2
-		tolerance = total * 0.20
+		tolerance = total * 0.15
 	)
 	got := make(map[string]float64, 2)
 	wantKeys := sets.NewString(fooName, barName)
-	for i := 0; i < total; i++ {
-		ri := RuntimeRequest(t, client, "http://"+name+".example.com/foo")
-		if ri == nil {
-			return
-		}
+	wg := pool.New(total)
```

Comment:
This should be something smaller. Maybe 8? It's the number of requests to send in parallel.